.github/workflows/pt-conda.yml

```diff
@@ -22,21 +22,23 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: [3.7]
-        # todo: add nightly versions
-        pytorch-version: [1.3, 1.4, 1.5, 1.6]  # , 1.7
+        pytorch-version: [1.3, 1.4, 1.5, 1.6, 1.7]
 
     # Timeout: https://stackoverflow.com/a/59076067/4521646
     timeout-minutes: 35
     steps:
     - uses: actions/checkout@v2
 
-    - name: Setup pyTorch
+    - name: Setup PyTorch nightly channel
+      if: matrix.pytorch-version == 1.7
+      run: |
+        python -c "fname = 'environment.yml' ; req = open(fname).read().replace('pytorch', 'pytorch-nightly', 1) ; open(fname, 'w').write(req)"
```

Comment:
`pytorch` is also a package name, so we hope that the channels ale always before packages...

Reply:
Yes, I am only replacing `pytorch` channel with `pytorch-nightly` channel

Reply:
it is fine, just saying let's be careful 🐰

Reply:
Okay